step-templates/pingdom-create-uptime-check.json

```diff
@@ -0,0 +1,242 @@
+{
+  "Id": "70ea4820-7d02-4015-a691-8c77b5ab14d5",
+  "Name": "Pingdom - Create Uptime Check",
+  "Description": "Creates Pingdom http check using [Create New Check API method](https://www.pingdom.com/resources/api#MethodCreate+New+Check).",
+  "ActionType": "Octopus.Script",
+  "Version": 27,
+  "CommunityActionTemplateId": null,
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Package",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptFileName": "#{Utilities.PingdomCreateUptimeCheckScriptPath}",
```

Comment:
Oh boy, forgot to change it. :D 
That's how we reference scripts in our own templates.